### DISCUSSION TITLE
Kubernetes and multiple replicas

### DISCUSSION THREAD

**Discussion by Drabuna** [btcpayserver/btcpayserver#3243] [Technical Support] [answered]
Hi,

Is anyone running btcpayserver on a Kubernetes cluster?
Is it possible to have multiple replicas up, or will it cause problems and I should run a single replica only?

Thanks!

**Comment by Jossec101** [comment]
Yes, has anyone tried scaling it? What about the events and concurrency of the data?

**Comment by Kukks** [comment, marked as answer]
Btcpay and nbxplorer are stateful services, do not run in a replicated
environment.
